src/Entity/BoxConfiguration.php

```diff
@@ -116,6 +116,11 @@ class BoxConfiguration
      */
     private $defaultLanguageCode = 'EN';
 
+    /**
+     * @ORM\Column(type="string", length=255)
```

Comment:
If the length is only 10 chars no need to make space for 255 chars!

Reply:
It leaves space for other choices of lengths.